### ISSUE TITLE
Non-null fields not assigned to in the constructors produce no warnings

### ISSUE THREAD

**Issue by GoogleCodeExporter** [typetools/checker-framework#42]
```
What steps will reproduce the problem?

class AaTest {
  Object nullObject;
  public Object getNull() { return nullObject; }
}

What is the expected output? What do you see instead?

Expected: a warning somewhere (getNull() returns 'null')
Actual: no warnings

What version of the product are you using? On what operating system?

Linux, checker v1.0.3a

Please provide any additional information below.

```

Original issue reported on code.google.com by `piotr.zi...@gmail.com` on 10 Dec 2009 at 8:18
